backend/dal/Migrations/20210126015055_v01.06.03.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Pims.Dal.Helpers.Migrations;
+using Pims.Dal.Helpers.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v010603 : SeedMigration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.DropIndex(
+                name: "IX_Parcels_Id_IsSensitive_AgencyId_ClassificationId_PID_PIN_AddressId_ProjectNumber_LandArea_Zoning_ZoningPotential",
+                table: "Parcels");
+
+            migrationBuilder.DropIndex(
+                name: "IX_Buildings_Id_IsSensitive_AgencyId_ClassificationId_AddressId_ProjectNumber_BuildingConstructionTypeId_BuildingPredominateUse~",
+                table: "Buildings");
+
+            migrationBuilder.DropColumn(
```

Comment:
You may have saved yourself some effort by just renaming the column and making it larger.
I'm wondering if we've introduced an issue with indexing now that `ProjectNumbers` is so large?

Reply:
I'm hoping the index won't take a performance hit since the actual column size is unlikely to change in most cases. Certainly initially after the migration runs the projectnumbers field will be roughly the same size in all cases.